adapter/src/main/java/com/twilio/kudu/sql/KuduEnumerable.java

```diff
@@ -490,11 +490,11 @@ public Schema getTableSchema() {
           .newScanTokenBuilder(calciteKuduTable.getKuduTable());
       // ReplicaSelection added to avoid querying leader always
       tokenBuilder.replicaSelection(ReplicaSelection.CLOSEST_REPLICA);
```

Comment:
could the replicationSelection to CLOSEST_REPLICA  also slow down the queries?

Reply:
I think `CLOSEST_REPLICA` is for drill/impala where they execute scans on the datanodes and so its possible to locate the compute on the datanode that hosts the tablet being queried.  
In our case it seems it will pick a random replica so that might help if a datanode does down. 
https://kudu.apache.org/apidocs/org/apache/kudu/client/ReplicaSelection.html